crypto/src/proof_of_work.rs

```diff
@@ -109,7 +109,7 @@ pub fn check_proof_of_work(data: &[u8], target: f64) -> PowResult {
 }
 
 fn check_proof_of_work_inner(data: &[u8], target_number: &BigUint) -> PowResult {
-    let hash = blake2b::digest_256(data);
+    let hash = blake2b::digest_256(data).map_err(|_| ())?;
```

Comment:
@akoptelov 
now when digest_256 returns Blake2bError,
I would add here also:
```
impl From<Blake2bError> for PowResult {
    fn from(...) -> Self {
...
    }
}
```

and then we can use: `blake2b::digest_256(data)?`